src/main/java/com/monumental/services/TagService.java

```diff
@@ -93,30 +93,11 @@ public Tag createTag(String name, List<Monument> monuments, Boolean isMaterial)
      * @param monument - Monument to remove the specified Tag from
      * @return Tag - The updated Tag with the specified Monument removed
      */
-    public Tag removeTagFromMonument(Tag tag, Monument monument) {
+    public Tag removeTagFromMonument(MonumentTag tag, Monument monument) {
         if (tag == null || monument == null) {
             return null;
         }
-
-        this.initializeAllLazyLoadedCollections(tag);
-
```

Comment:
Have you tested if removing this effects other parts of the system. Not 100% sure what that previous logic was for.

Reply:
@dn3531 I changed this method so now it removes monument tags from monuments directly. So because it uses MonumentTag now rather than Tag, this method doesn't do anything. I also updated the functionality in MonumentService#updateMonumentTags() so that it doesn't break the existing functionality.